metpy/calc/tests/test_basic.py

```diff
@@ -261,13 +261,13 @@ def test_coriolis_force():
 def test_add_height_to_pressure():
     """Test the pressure at height above pressure calculation."""
     pressure = add_height_to_pressure(1000 * units.hPa, 877.17421094 * units.meter)
-    assert_almost_equal(pressure, 900 * units.hPa, 5)
+    assert_almost_equal(pressure, 900 * units.hPa, 4)
```

Comment:
I like having it one place past where we are comparing for easy visual reference, but that's probably just the CDO talking.